FSJ_django20_project/FSJ/migrations/0002_award.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.0.2 on 2018-02-22 04:00
+
```

Comment:
To date I haven't been uploading the migration files since I (and our clients) start with no database and can generate migrations prior to migrating unless there's some manual modification of a migration file that would require us to actually save it. Is something in these migrations other than system generated content?

Reply:
Nope, this was uploaded by accident, I'll remove it